### PR TITLE
Add core weekly 3 of 2020

### PR DESCRIPTION
**Pull request by matks**
To be published on the 23th of January

### CODE REVIEW

**Comment by matks** [comment]
@ttoine @eternoendless Please validate this PR (mainly for the "general messages" part) 😄

**Review by ttoine — APPROVED** [review]

**Comment by ttoine** [comment]
@eternoendless I let you have a final check: the first paragraph is really about technical choices and I am not the good one to be sure there is no mistake.

**Comment by matks** [comment]
⚠️ if this is merged later than today, please modify publication date !!!

File: news/_posts/core-weekly/2020-01-23-coreweekly-week-03-2020.md [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+
+Dear Developers,
+
+There is a lot of things going on this week !
+
+The first one is obviously about PrestaShop 1.7.6.3 being available. This maintenance release fixes 16 issues reported on versions 1.7.6.0, 1.7.6.1 and 1.7.6.2. Please [read the release note](https://build.prestashop.com/news/prestashop-1-7-6-3-maintenance-release/) for more informations.
+
+Secondly, we have created the `1.7.7.x` branch which will carry the changes being released with 1.7.7.0. So from now on, the work carried out on `develop` branch will be released with 1.7.8.
+
+Finally, after asking you what was your opinion about [committing generated files](https://build.prestashop.com/news/open-question-not-commiting-assets-anymore/) and receiving positive feedback for the change, we have implemented it into the [develop branch](https://github.com/PrestaShop/PrestaShop/pull/16670). We have also [updated the developer documentation](https://github.com/PrestaShop/docs/pull/418) accordingly in order to provide prestashop users all the necessary informations to be able to generate these files themselves.
```
Comment (eternoendless) [inline news/_posts/core-weekly/2020-01-23-coreweekly-week-03-2020.md]:
```suggestion
Finally, after asking you what was your opinion about [no longer committing generated files](https://build.prestashop.com/news/open-question-not-commiting-assets-anymore/) and receiving positive feedback for the change, we have implemented it into the [develop branch](https://github.com/PrestaShop/PrestaShop/pull/16670). We have also [updated the developer documentation](https://github.com/PrestaShop/docs/pull/418) accordingly in order to provide developers with all the necessary information to be able to generate these files on their own.
```

**Comment by matks** [comment]
Thanks for review

**Pull request merged**